browser-test/src/applicant/questions/radio_button.test.ts

```diff
@@ -93,6 +100,35 @@ test.describe('Radio button question for applicant flow', () => {
       await validateScreenshot(page, 'radio-button-errors')
     })
 
+    test(
+      'validate screenshot with north star flag enabled',
+      {tag: ['@northstar']},
+      async () => {
+        const {page, applicantQuestions} = ctx
+        await enableFeatureFlag(page, 'north_star_applicant_ui')
+        await applicantQuestions.applyProgram(programName)
+
+        await test.step('Screenshot', async () => {
```

Comment:
```suggestion
        await test.step('Screenshot without errors', async () => {
```

Makes it more clear what the screenshot is for

Reply:
done